packages/social/src/abstract-provider.service.spec.ts

```diff
@@ -19,12 +21,14 @@ import {
 import {
   AbstractProvider,
   AuthorizationError,
+  InvalidCodeChallengeError,
   InvalidStateError,
   SocialTokens,
   TokenError
 } from './abstract-provider.service';
 
 const STATE_COOKIE_NAME = 'oauth2-state';
+const CODE_VERIFIER_NAME = 'oauth2-code-challenge';
```

Comment:
One test is failing: https://github.com/FoalTS/foal/runs/6262307685?check_suite_focus=true. I think it's because the cookie name is not the same in the test and in the tested code.

Reply:
You are right... i've made a commit without saving last changes. Sorry!!!